### PR TITLE
Rounded framerate to 3 decimal places per spec

### PR DESCRIPTION
**Pull request by thenewguy**
Fixes issue #216

### CODE REVIEW

**Comment by leandromoreira** [comment]
hey @thenewguy can you add some tests to ensure this behavior?

* a integer framerate
* a double digits float framerate
* a 5 digits float framerate

tell me if you need any assistance.

**Comment by mauricioabreu** [comment]
Agree with @leandromoreira 

Nice job anyways @thenewguy

**Comment by thenewguy** [comment]
> hey @thenewguy can you add some tests to ensure this behavior?
> 
> * a integer framerate
> * a double digits float framerate
> * a 5 digits float framerate
> 
> tell me if you need any assistance.

Integer and float tested already at - VARIANT_PLAYLIST_WITH_FRAME_RATE

I will add a test that reads a long 5 digit one in and strings it back to 3 digits

File: tests/test_parser.py [hunk 1]
```diff
@@ -400,6 +400,11 @@ def test_master_playlist_with_frame_rate():
     assert 60 == playlists_list[2]['stream_info']['frame_rate']
     assert 12.5 == playlists_list[3]['stream_info']['frame_rate']
 
+def test_master_playlist_with_unrounded_frame_rate():
+    data = m3u8.parse(playlists.VARIANT_PLAYLIST_WITH_ROUNDABLE_FRAME_RATE)
+    playlists_list = list(data['playlists'])
+    assert 12.54321 == playlists_list[0]['stream_info']['frame_rate']
```
Comment (leandromoreira) [inline tests/test_parser.py:406]: Shouldn't be already fixed 3 digits here?

```python
     assert 12.543 == playlists_list[0]['stream_info']['frame_rate'] 
```

Comment (thenewguy) [reply tests/test_parser.py:406]: This verifies we can parse a playlist containing a frame rate longer than 3 digits - it isn't rounded until it is converted back to string with dumps

**Pull request merged**